meta-ros2-humble/recipes-bbappends/menge-vendor/menge-vendor/0001-CMakeLists-search-for-tinyxml2-instead-of-tinyxml.patch

```diff
@@ -0,0 +1,27 @@
+From be9a3828d3ce82e53434a9185159b4373ebc8b7c Mon Sep 17 00:00:00 2001
```

Comment:
do we still need this patch? `tinyxml` is present

Reply:
Yes, we still need this patch.  According to the comment in the patch says "libtinyxml from meta-oe doesn't provide .pc file"

rwoolley at ala-lpggp3 in /ala-lpggp31/rwoolley/meta-ros-humblepr1/tmp-glibc/work/core2-64-oe-linux
$ find libtinyxml* -name 'tiny*.pc'
libtinyxml2/9.0.0-r0/sysroot-destdir/usr/lib/pkgconfig/tinyxml2.pc
libtinyxml2/9.0.0-r0/packages-split/libtinyxml2-dev/usr/lib/pkgconfig/tinyxml2.pc
libtinyxml2/9.0.0-r0/build/tinyxml2.pc
libtinyxml2/9.0.0-r0/image/usr/lib/pkgconfig/tinyxml2.pc
libtinyxml2/9.0.0-r0/package/usr/lib/pkgconfig/tinyxml2.pc

rwoolley at ala-lpggp3 in /ala-lpggp31/rwoolley/meta-ros-humblepr1/tmp-glibc/work/core2-64-oe-linux
$ ls libtinyxml/2.6.2-r5/image/usr/lib/
libtinyxml.so  libtinyxml.so.2.6.2

libtinyxml does not provide a pkgconfig file, but tinyxml2 does.